tests/results/TextRangeTest.py

```diff
@@ -35,7 +36,8 @@ def test_properties(self):
 
     def test_from_values(self):
         # Check if invalid ranges still fail.
-        with self.assertRaises(ValueError):
+        END_LT_START_RE = "End position can't be less than start position"
```

Comment:
Add `END_LT_START_RE` and `JOIN_INSTANCEOF_RE` as class level attributes, as suggested by @jayvdb and not at function level to remove redundant declaration in multiple functions.